components/admin_console/admin_definition.jsx

```diff
@@ -975,6 +975,32 @@ const AdminDefinition = {
                         onConfigSave: (displayVal) => displayVal * MEBIBYTE,
                         isDisabled: it.not(it.userHasWritePermissionOnResource(RESOURCE_KEYS.ENVIRONMENT.FILE_STORAGE)),
                     },
+                    {
+                        type: Constants.SettingsTypes.TYPE_BOOL,
+                        key: 'FileSettings.ExtractContent',
+                        label: t('admin.image.extractContentTitle'),
+                        label_default: 'Enable document search by content:',
+                        help_text: t('admin.image.extractContentDescription'),
+                        help_text_markdown: true,
+                        help_text_default: 'When enabled, supported document types are searchable by their content. Search results for existing documents may be incomplete [until a data migration is executed](!https://www.mattermost.com/file-content-extraction).',
+                        isDisabled: it.any(
+                            it.not(it.userHasWritePermissionOnResource('environment')),
```

Comment:
Heads up that the keys here were changed and moved to constants. (Happened over on the other feat. branch too 😛)

```suggestion
                            it.not(it.userHasWritePermissionOnResource(RESOURCE_KEYS.ENVIRONMENT.FILE_STORAGE)),
```

<img width="880" alt="CleanShot 2021-04-08 at 20 46 33@2x" src="https://user-images.githubusercontent.com/11724372/114117124-9d405280-98ab-11eb-9ef0-cbf01101719d.png">

Reply:
Done